### PR TITLE
Add Rigetti Provider Documentation

### PR DESCRIPTION
**Pull request by kalzoo**
This includes:

- Main Rigetti documentation
- Support documentation
- Pricing information

### CODE REVIEW

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[41402b0](https://github.com/rigetti/microsoft-azure-quantum-docs/commits/41402b03625ed267d0b873dbf7266839b53fd6d4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md) | :bulb:Suggestion | | [Details](#user-content-514026d2fff7346a88e1f71d3a400d98-507d22ab)
[articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md) | :bulb:Suggestion | | [Details](#user-content-e5c5a2b0df7d907df261bc8920be51c3-507d22ab)
[articles/includes/pricing-qc.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/includes/pricing-qc.md) | :white_check_mark:Succeeded | |

<a id="514026d2fff7346a88e1f71d3a400d98-507d22ab"></a>
### [articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md)
 - **Line 4, Column 12**: **[Suggestion: ms-author-invalid - [See documentation](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/ms-author-invalid?branch=main)]** `````Invalid value for 'ms.author', 'kalzoo' is not a valid Microsoft alias.`````

<a id="e5c5a2b0df7d907df261bc8920be51c3-507d22ab"></a>
### [articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md)
 - **Line 4, Column 12**: **[Suggestion: ms-author-invalid - [See documentation](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/ms-author-invalid?branch=main)]** `````Invalid value for 'ms.author', 'kalzoo' is not a valid Microsoft alias.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202209010301154347-563/BuildReport?accessString=03cda9eec44d197fcba46a993475924ba35f5251b75da470b98d420b05d26faf).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[0fda156](https://github.com/rigetti/microsoft-azure-quantum-docs/commits/0fda1561709219fb109bce05196f54b5ba85719a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md) | :warning:Warning | | [Details](#user-content-514026d2fff7346a88e1f71d3a400d98-a54a02f4)
[articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md) | :warning:Warning | | [Details](#user-content-e5c5a2b0df7d907df261bc8920be51c3-a54a02f4)
[articles/includes/pricing-qc.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/includes/pricing-qc.md) | :white_check_mark:Succeeded | |

<a id="514026d2fff7346a88e1f71d3a400d98-a54a02f4"></a>
### [articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md)
 - **Line 2, Column 1**: **[Warning: ms-author-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-author-missing?branch=main)]** `````Missing required attribute: 'ms.author'. Add the current author's Microsoft alias.`````

<a id="e5c5a2b0df7d907df261bc8920be51c3-a54a02f4"></a>
### [articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md)
 - **Line 2, Column 1**: **[Warning: ms-author-missing - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-author-missing?branch=main)]** `````Missing required attribute: 'ms.author'. Add the current author's Microsoft alias.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202209010304275347-563/BuildReport?accessString=dedd52a4467fbde4da3b6f1c087547a4a1ae94c92803ad411f545212994e7b32).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by kalzoo** [comment]
The 🤖 doesn't like having no `ms.author` nor having my name in there, and clicking on the link results in this: 
image

@tedhudek, lmk what you'd like in that header field.

**Comment by Jak-MS** [comment]
@MicrosoftDocs/quantum_docs_writers @tedhudek - 
- Can you review this PR?
- Note that since it contains new content, the PR would need to be moved to the private repo and someone on the quantum docs team would need to take ownership.


IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/provider-rigetti.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+---
+author: kalzoo
```
Comment (Bradben) [inline articles/provider-rigetti.md:2]:
```suggestion
author: kalzoo
ms.author: brbenefield
```

File: articles/provider-support-rigetti.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+author: kalzoo
```
Comment (Bradben) [inline articles/provider-support-rigetti.md:2]:
```suggestion
author: kalzoo
ms.author: brbenefield
```

**Comment by Bradben** [comment]
@kalzoo - Thanks for helping out with the Rigetti content! I updated the metadata so you shouldn't be getting that warning anymore. I'll be editing the content to get it ready for publishing but ping me if you have any questions along the way. 

One other article that might need updating is this page - https://docs.microsoft.com/en-us/azure/quantum/quantum-computing-target-profiles. Can you review and see what applies to Rigetti or what new content may be needed? 

Thanks - Brad

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[32b52fd](https://github.com/rigetti/microsoft-azure-quantum-docs/commits/32b52fd6024217f7e3865860dc4007284bb8a91a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/includes/pricing-qc.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/includes/pricing-qc.md) | :white_check_mark:Succeeded | |
[articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md) | :white_check_mark:Succeeded | |
[articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202209061627319437-563/BuildReport?accessString=a8b5ede51e117ecf1cfe03e14992417421c02bb9158028fe01090fea6cd980e6).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[f82d16a](https://github.com/rigetti/microsoft-azure-quantum-docs/commits/f82d16a115cd9e9a2a7518b425c73a30f5092731)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/includes/pricing-qc.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/includes/pricing-qc.md) | :white_check_mark:Succeeded | |
[articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md) | :white_check_mark:Succeeded | |
[articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md) | :white_check_mark:Succeeded | |
[articles/quantum-computing-target-profiles.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/quantum-computing-target-profiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202209062022218260-563/BuildReport?accessString=810700b8177e40296aecdc2603f74fc8ad8261b073b365bafaf3cb1213eff2ce).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: articles/provider-rigetti.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+ 
+All targets also take the optional `count` integer parameter for defining the number of shots to run. If omitted, the program will only be run once.
+
+The Rigetti provider makes the following targets available:
+
+### Simulators
+
+#### QVM
+
+[QVM is an open-source simulator][QVM] for [Quil]. The `rigetti.sim.qvm` target accepts a [Quil] program as text and runs that program on [QVM] hosted in the cloud, returning simulated results.
```
Comment (Bradben) [inline articles/provider-rigetti.md:37]:
```suggestion
[QVM] is an open-source simulator for [Quil]. The `rigetti.sim.qvm` target accepts a Quil program as text and runs that program on QVM hosted in the cloud, returning simulated results.
```

File: articles/provider-rigetti.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+In addition to the Azure Quantum Credits plan, Rigetti offers a pay-as-you-go plan for live quantum processors, so you pay only for what you use.
+
+## Input Format
+
+### Quil
+
+All Rigetti targets accept the `rigetti.quil.v1` input format, which is the text of a [Quil] program. By default, the program will be compiled using [quilc] before running. However, [quilc] does not support the pulse level control features ([Quil-T]), so if you want to use those features you must provide a [Native Quil] program (also containing [Quil-T]) as input and specify the `skipQuilc` input parameter.
+
+To make constructing a [Quil] program easier, you can use [`pyQuil`] along with the [`pyquil-for-azure-quantum`] package. Without this package, [`pyQuil`] can be used to _construct_ Quil programs but not submit them to Microsoft Azure.
```
Comment (Bradben) [inline articles/provider-rigetti.md:85]:
```suggestion
All Rigetti targets accept the `rigetti.quil.v1` input format, which is the text of a [Quil] program. By default, the program will be compiled using [quilc] before running. However, quilc does not support the pulse level control features ([Quil-T]), so if you want to use those features you must provide a [Native Quil] program (also containing Quil-T) as input and specify the `skipQuilc` input parameter.

To make constructing a Quil program easier, you can use [`pyQuil`] along with the [`pyquil-for-azure-quantum`] package. Without this package, `pyQuil` can be used to _construct_ Quil programs but not submit them to Azure Quantum.
```

File: articles/provider-rigetti.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+# In this case, because the type of the register is BIT, the type will be integer and the value either 0 or 1
+assert isinstance(ro_data_first_shot[0], int)
+assert ro_data_first_shot[0] == 1 or ro_data_first_shot[0] == 0
+
+# Let's print out all the data
+print(f"Data from '{readout}' register:")
+for i, shot in enumerate(data_per_shot):
+    print(f"Shot {i}: {shot}")
+```
+
```
Comment (Bradben) [inline articles/provider-rigetti.md:193]:
```suggestion
***

```

File: articles/provider-support-rigetti.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ms.service: azure-quantum
+ms.subservice: computing
+ms.topic: reference
+title: Support Policy for Rigetti in Azure Quantum
+uid: microsoft.quantum.providers.rigetti.support
+---
+
+# Support Policy for Rigetti in Azure Quantum
+
+This article describes the Microsoft support policy that applies when you use the Rigetti provider in Azure Quantum. The article applies to any of the targets under this provider.
```
Comment (Bradben) [inline articles/provider-support-rigetti.md:15]:
```suggestion
This article describes the Microsoft support policy that applies when you use the Rigetti provider in Azure Quantum. The article applies to any of the [targets](xref:microsoft.quantum.providers.rigetti#targets) offered by the Rigetti provider.
```

File: articles/provider-support-rigetti.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+If you're using the Rigetti provider and experience unexpected issues, you can contact the Azure Support team for help
```
Comment (Bradben) [inline articles/provider-support-rigetti.md:17]:
```suggestion
If you're using the Rigetti provider and experience unexpected issues, you can contact the Azure Support team for help.
```

File: articles/provider-support-rigetti.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Rigetti will always attempt to resolve issues with the Microsoft Azure integration as quickly as practical. Some issues will take longer to address than others, but the tables below indicate the time you may expect for an initial response from Rigetti after an issue report.
+
+| Severity   | Definition        | Response SLA        |
+|------------|-------------------|---------------------|
+| High | Failure in connectivity; complete performance degradation | 1 business day |
+| Medium | Performance degradation | 2 business days |
+| Low | Application-related concerns and questions | 5 business days |
+
+"Business days" are defined as Monday through Friday, not counting US holidays. Example: if a medium-severity incident is reported on a Tuesday, a response will be returned by the end of the business day on Thursday, if the intervening days are also business days.
```
Comment (Bradben) [inline articles/provider-support-rigetti.md:35]:
```suggestion
*Business days* are defined as Monday through Friday, not counting US holidays. For example, if a medium-severity incident is reported on a Tuesday, a response will be returned by the end of the business day on Thursday, if the intervening days are also business days.
```

File: articles/provider-rigetti.md [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+Should you use Quil or QIR? It comes down to your end use case. QIR is more accessible for many users; Quil is more powerful today.
+
+If you're using Qiskit, Q#, or another toolkit that supports QIR generation, and your application works on Rigetti targets via Azure Quantum, then QIR is right for you! QIR has a rapidly evolving specification, and Rigetti will continue to increase support for more advanced QIR programs as time passes. What can't be compiled today may well compile tomorrow.
+
+On the other hand, Quil programs express the full set of functionality available to users of Rigetti systems from any platform including Microsoft Azure. If you're looking to tailor the decomposition of your quantum gates or write programs at the pulse level, then you'll want to work in Quil, because those capabilities are not yet available through QIR.
+
+## Example Using [`pyquil-for-azure-quantum`]
+
+The easiest way to submit Quil jobs is using the [`pyquil-for-azure-quantum`] package, as it allows you to leverage the tools and documentation of the [`pyQuil`] library.
```
Comment (Bradben) [inline articles/provider-rigetti.md:101]:
```suggestion
The easiest way to submit Quil jobs is using the [`pyquil-for-azure-quantum`] package, as it allows you to leverage the tools and documentation of the [`pyQuil`] library.

You can also construct Quil programs manually and submit them using the `azure-quantum` package directly.
```

**Review by Bradben — CHANGES_REQUESTED** [review]
Minor copy edits and formatting standards - it looks good, thanks!

**Comment by Bradben** [comment]
@kalzoo - do you have time to look at and commit these changes today? Thanks!

File: articles/provider-rigetti.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+
+[**Rigetti's**](https://rigetti.com) pricing model is simple.
+
+Rigetti's simulator, [**Quantum Virtual Machine (QVM)**] ([https://pyquil-docs.rigetti.com/en/1.9/qvm.html]) is free for all users.
+
+Rigetti's live quantum processors, Aspen-11 and Aspen-M-2, are billed by job execution time only: $0.02 per 10 millisecond increment. There is no added charge per job, per shot, or per gate.
+
+All new Azure Quantum customers benefit from $500 (USD) free Azure Quantum credits to use specifically for Rigetti target(s). To learn more about credits, see [Azure Quantum Credits](xref:microsoft.quantum.credits).
+
+In addition to the Azure Quantum Credits plan, Rigetti offers a pay-as-you-go plan for live quantum processors, so you pay only for what you use.
```
[the remark this answers is not in the document]
Comment (kalzoo) [reply articles/provider-rigetti.md:77]:
```suggestion
To see Rigetti's billing plan, visit [Azure Quantum pricing](xref:microsoft.quantum.providers-pricing).
```

File: articles/provider-rigetti.md [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+
+- Job Type: `Simulation`
+- Data Format: `rigetti.quil.v1`
+- Target ID: `rigetti.sim.qvm`
+- Target Execution Profile: [No Control Flow](xref:microsoft.quantum.target-profiles)
+- Pricing: Free ($0)
+
+### Quantum Computers
+
+All of Rigetti's [publicly available QPUs](https://qcs.rigetti.com/qpus) are available through Azure Quantum. QPU targets optionally take a `skipQuilc` parameter for skipping the default compilation stage. Note that this list is subject to frequent change over time without advance notice.
```
[the remark this answers is not in the document]
Comment (kalzoo) [reply articles/provider-rigetti.md:47]:
```suggestion
All of Rigetti's [publicly available QPUs](https://qcs.rigetti.com/qpus) are available through Azure Quantum. Note that this list is subject to frequent change over time without advance notice.
```

File: articles/provider-rigetti.md (hunk 4 above)
Comment (kalzoo) [reply articles/provider-rigetti.md:85]:
```suggestion
All Rigetti targets accept the `rigetti.quil.v1` input format, which is the text of a [Quil] program. By default, the program will be compiled using [quilc] before running. However, quilc does not support the pulse level control features ([Quil-T]), so if you want to use those features you must provide a [Native Quil] program (also containing Quil-T) as input and specify the input parameter `skipQuilc: true`.
	
To make constructing a Quil program easier, you can use [`pyQuil`] along with the [`pyquil-for-azure-quantum`] package. Without this package, `pyQuil` can be used to _construct_ Quil programs but not submit them to Azure Quantum.
```

**Comment by kalzoo** [comment]
@Bradben done - accepted most as-is, tweaked a couple. Thanks!

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[39cb3ee](https://github.com/rigetti/microsoft-azure-quantum-docs/commits/39cb3ee375cebca8ac1b821ab8ee7b329c9d6e8e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/includes/pricing-qc.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/includes/pricing-qc.md) | :white_check_mark:Succeeded | |
[articles/provider-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-rigetti.md) | :white_check_mark:Succeeded | |
[articles/provider-support-rigetti.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/provider-support-rigetti.md) | :white_check_mark:Succeeded | |
[articles/quantum-computing-target-profiles.md](https://github.com/rigetti/microsoft-azure-quantum-docs/blob/feat/rigetti-provider/articles/quantum-computing-target-profiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202209071918518409-563/BuildReport?accessString=f900dc0fe6506a834810163da161202e7d29aafcb3acd18f78aa6414afbc74c6).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by Bradben — APPROVED** [review]

**Comment by Bradben** [comment]
Moved to private repo for testing and verification.

**Pull request closed without merging**